exporter/awscloudwatchlogsexporter/metadata.yaml

```
@@ -7,3 +7,13 @@ status:
   distributions: [contrib, observiq, liatrio]
   codeowners:
     active: [boostchicken, bryan-aguilar, rapphil]
+
+tests:
+  config:
+    endpoint: http://localhost:0
+    region: local
+    sending_queue:
+      enabled: false
+    retry_on_failure:
+      enabled: false
```

```suggestion
    log_group_name: "testing-logs"
    log_stream_name: "testing-integrations-stream"
```

Technically this is the smallest config required for the exporter